Bottom: 5c6ef04d24e2de8e44294706251761c6ec254ff0
Top:    ca3a6110b7c48397a33bd5a590f1e24c83f64734
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 21:18:43 -0400

Refresh of dma-module

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 8060b95..dc79bdf 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -1,5 +1,5 @@
 config SND_SOC_OF_SIMPLE
-	bool
+	tristate
 	
 config SND_MPC52xx_DMA
 	tristate
